datafusion/src/sql/planner.rs

```diff
@@ -2826,6 +2818,16 @@ mod tests {
         quick_test(sql, expected);
     }
 
+    #[test]
+    fn empty_over_dup_with_alias() {
+        let sql = "SELECT order_id oid, MAX(order_id) OVER () max_oid, MAX(order_id) OVER () max_oid_dup from orders";
+        let expected = "\
+        Projection: #orders.order_id AS oid, #MAX(orders.order_id) AS max_oid, #MAX(orders.order_id) AS max_oid_dup\
+        \n  WindowAggr: windowExpr=[[MAX(#orders.order_id)]]\
```

Comment:
this is pretty cool that the planner has identified the redundancy and only computes `MAX` once 👍

Reply:
not only the planner, the builder also does this for those who use the dataframe apis directly ;)